hack/vpc-resource-controller-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: amazon-vpc-cni
+  namespace: kube-system
+data:
+  enable-windows-ipam: "true"
```

Comment:
According to the `eksctl` docs (https://eksctl.io/usage/windows-worker-nodes/):

> eksctl will automatically patch the ConfigMap to enable Windows IP address management when a Windows nodegroup is created.

Is this file even necessary?

Reply:
This file is necessary, tested the behavior without applying the ConfigMap and we run into:
```
Warning  FailedCreatePodSandBox  11m                  kubelet            Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "4b2e899fd11e35f9f5ca4423a0a2d34d42e0bbf2c0034c59fce5ff7c63da1989": plugin type="vpc-shared-eni" name="vpc" failed (add): failed to parse Kubernetes args: pod does not have label vpc.amazonaws.com/PrivateIPv4Address
```